libs/math/src/math/linalg/blas/gemm_tt_vector.cpp

```diff
@@ -110,6 +110,14 @@ template class Blas<float, Signature(_C <= _alpha, _A, _B, _beta, _C),
                     Computes(_C <= _alpha * T(_A) * T(_B) + _beta * _C),
                     platform::Parallelisation::VECTORISE>;
 
+template class Blas<
```

Comment:
and here?

Reply:
same, no support for fp128 in vectorisation.